http/1.9/modules/ROOT/pages/http-connector-config-acb.adoc

```diff
@@ -0,0 +1,143 @@
+|Response validator | Configures default error handling for the request. This attribute is not currently enabled in the configuration UI. Configure it in the XML editor. See xref:http-documentation.adoc#SuccessStatusCodeValidator[Success Status Code Validator] and xref:http-documentation.adoc#FailureStatusCodeValidator[Failure Status Code Validator].
+|Target Variable | Name of the variable on which the operation's output is placed.
+|Target Value | An expression that's evaluated against the operation's output. The outcome of that expression is stored in the target variable.
+| Error Mappings | This attribute is not currently enabled. 
+|Reconnection Strategy |This attribute is not currently enabled in the configuration UI. Configure it in the XML editor. See xref:http-documentation.adoc#reconnect[Reconnect] and xref:http-documentation.adoc#reconnect[Reconnect Forever]. 
+|===
+
+[[configure-load-static-resource]]
+=== Configure Attributes for Load Static Resource
+
+The static resource serves static content to use with HTTP. The request path is used to look up the resource. 
+
+To configure the attributes for HTTP *Load static resource* in the Anypoint Code Builder UI:
+
+. Click the *Load static resource* component on the canvas.
+. In the *General* tab, configure the following attributes:
++
+[%header%autowidth.spread]
+|===
+| UI Field | Description 
+|Resource base path |The base path of the resource that serves up the content, for example, `/Users/maxthemule/` resources.
+|Default file | The default file to serve when a directory is specified. The default is `index.html`.
+|Attributes | The HTTP request attributes defined in the HTTP Listener source to check the required resources. 
+|===
+. Select the *Advanced* tab to configure the following attributes: 
++
+[%header%autowidth.spread]
+|===
+| UI Field | Description 
+|Output MIME type | MIME type of the payload this component outputs.
+| Parameters |Appends a key-value pair for a reader property to the value of the Output MIME attribute. Multiple key-value pairs are allowed. 
+| Output encoding | Encoding of the payload this component outputs.
+|Target Variable | Name of the variable that stores the operation's output.
+|Target Value |Expression that evaluates the operation's output. The expression outcome is stored in the target variable.
+|Error Mappings | This attribute is not currently enabled.
+|===
+
+== Configure Authentication for the Connector
+
+The Anypoint Connector for HTTP (HTTP Connector) *Request* operation supports connecting a Mule client app to a service that requires any of the following types of authentication:
+
+* Basic Authentication
+* Digest Authentication
+* NTLM Authentication
+* OAuth2 Authorization Code Grant Type Authentication
+* <<oauth2-client-authentication,OAuth2 Client Credentials Grant Type Authentication>>
+[[testing-http-listener-connection]]
```

Comment:
linebreak here so the "Test an HTTP Listener Connection" header renders properly